Animations/src/core/Window.cpp

```diff
@@ -16,12 +16,6 @@ namespace MathAnim
 	Window::Window(int width, int height, const char* title, WindowFlags flags)
 		: width(width), height(height), title(title)
 	{
-		if (GLVersion.major < 3 || (GLVersion.major >= 3 && GLVersion.minor < 1)) 
```

Comment:
I honestly just removed that check because after upgrading to GLAD version 2, there is no global `GLVersion` structure anymore. I agree that this check should be added back in, but I'm not sure where (it didn't seem to be in the right place there)